src/shared/components/person/person-listing.tsx

```diff
@@ -88,7 +88,10 @@ export class PersonListing extends Component<PersonListingProps, any> {
           !this.props.person.banned &&
           showAvatars() && (
             <PictrsImage
-              src={avatar ?? "/static/assets/icons/icon-96x96.png"}
+              src={
+                avatar ??
+                `/static/${process.env.COMMIT_HASH}/assets/icons/icon-96x96.png`
```

Comment:
Can you make this into a `bundleFolder()` or `bundleDir()` function? That way we won't have to trace all these down again potentially in the future.

Reply:
I'll make the change during my lunch break.

Reply:
Made the changes.